angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/weather.html

```diff
@@ -1,3 +1,14 @@
+
+    <div class="weather-dropdown" dropdown is-open="status.isopen">
+      <button id="single-button" type="button" class="btn btn-primary rounded" dropdown-toggle ng-disabled="disabled">
+       <i class="fa fa-wrench "></i>
+      </button>
+      <ul class="dropdown-menu" role="menu" aria-labelledby="single-button">
+        <li role="menuitem"><a href="" ng-click="showMetric=false; changeToF()">Imperial</a></li>
```

Comment:
Looks like ["Imperial" may not be the best way to refer to this](https://en.wikipedia.org/wiki/United_States_customary_units).  Consider instead "US customary" or maybe even just get right to the point and call these "Fahrenheit" and "Celsius", even if they imply different presentation of things other than temperature too.